src/scripts.js

```diff
@@ -31,27 +41,54 @@ availableCards.addEventListener('click', function() {
 })
 selectedRoom.addEventListener('click', function() {
   addNewBooking(event);
-})
-selectedRoom.addEventListener('click', function() {
   closeSelectedRoom(event);
 })
-
+loginBtn.addEventListener('click', logInUser);
+newUser.addEventListener('click', changeUser);
 
 export function loadPageInfo(){
   apiCalls.getData()
     .then(response => {
       guestData = response[0].customers;
-      rooms = response[1].rooms;
-      bookings = response[2].bookings;
-      guest = new Guest(guestData[5]);
+      roomsData = response[1].rooms;
+      bookingsData = response[2].bookings;
+      instantiateRooms(roomsData);
+      instantiateBookings(bookingsData)
```

Comment:
You may need to fetch data again and re-instantiate the hotel after a successful booking...